### DISCUSSION TITLE
Export WhatsApp chat considering only a certain period of time

### DISCUSSION THREAD

**Discussion by gisms** [sepinf-inc/IPED#2133] [Q&A] [answered]
Hello!!

I believe it's not possible, but maybe I'm not seeing something:

... Would it be possible to export just a part of a WhatsApp chat, considering only a certain period of time?

**Comment by paulobreim** [comment]
In indexing it is not possible, but then you can select in ipedsearch, only the conversations that are of interest, and then generate a subset of the case that will only have what you selected.

**Comment by lfcnassif** [comment, marked as answer]
> ... Would it be possible to export just a part of a WhatsApp chat, considering only a certain period of time?

It's not possible to export just a part of the WhatsApp chat HTML report. But it is possible to export a specific time period of interest in CSV format if you select the "Instant Messages" category, right click on the results table and export properties of selected messages.
